LottoDiary/LottoDiary/Sources/Present/ViewController.swift

```diff
@@ -18,6 +18,14 @@ class ViewController: UIViewController {
         v.layer.borderColor = UIColor.designSystem(.mainOrange)?.cgColor
         return v
     }()
+    
+    let codeReviewLabel: UILabel = {
+        let v = UILabel()
+        v.text = "코드리뷰설명 PR을 위한 라벨입니다"
+        v.font = .systemFont(ofSize: 15, weight: .bold)
+        v.textColor = .red
```

Comment:
```suggestion
        v.textColor = .green
```
빨간색보다 녹색이 더 눈이 편안해 보일 것 같습니다.